docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+- [Expectations and Expectation Suite tutorials](/docs/guides/expectations/create_manage_expectations_lp)
+
+### Data Assistants
+
+A Data Assistant is a utility that automates the process of building Expectations by asking questions about your data, gathering information to describe what is observed, and then presenting Metrics and proposed Expectations based on the answers it has determined.  This can greatly accelerate the process of creating Expectations for the provided data.
+
+- [Data Assistant tutorials](/docs/guides/expectations/profilers_data_assistants_lp)
+
+### Custom Expectations
+
+GX comes with a built-in library of more than 50 common expectations.  However, you can also create custom Expectations if you need something more specialized for your data validation.  You can even use custom Expectations that have been contributed by other GX community members by installing them as Plugins.
+
+- [Tutorials for creating custom Expectations](/docs/guides/expectations/custom_expectations_lp)
+- [Tutorials for using custom Expectations](/docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations)
+
+## Checkpoints
+
+![Checkpoint overview](./images_overview/checkpoint_flowchart.png)
+
+A Checkpoint is the primary means for validating data in a production deployment of GX. Checkpoints provide an abstraction for bundling a Batch (or Batches) of data with an Expectation Suite (or several), and then running those Expectations against the paired data.
+
+- [Checkpoint tutorials](/docs/guides/validation/checkpoints/checkpoint_lp)
+
+### Validation Results
+
+The Validation Results returned by GX tell you how your data corresponds to what you expected of it. You can view this information in the Data Docs that are configured in your Data Context. Evaluating your Validation Results helps you identify issues with your data. If the Validation Results show that your data meets your Expectations, you can confidently use it.
+
+- [Validation Result further reading](/docs/terms/validation_result)
+
+### Actions
+
+One of the most powerful features of Checkpoints is that you can configure them to run Actions. The Validation Results generated when a Checkpoint runs determine what Actions are performed. Typical use cases include sending email, Slack, or custom notifications. Another common use case is updating Data Docs sites. Actions can be used to do anything you are capable of programming in Python. Actions are a versatile tool for integrating Checkpoints in your pipeline's workflow.
```

Comment:
```suggestion
One of the most powerful features of Checkpoints is that you can configure them to run Actions. The Validation Results generated when a Checkpoint runs determine what Actions are performed. Typical use cases include sending email, Slack messages, or custom notifications. Another common use case is updating Data Docs sites. Actions can be used to do anything you are capable of programming in Python. Actions are a versatile tool for integrating Checkpoints in your pipeline's workflow.
```

Reply:
suggestion implemented; it'll be in the next commit.